README.md

```diff
@@ -1 +1,84 @@
-# java-spiffe
\ No newline at end of file
+# JAVA-SPIFFE library
+
+## Overview
+
+The JAVA-SPIFFE library provides functionality to fetch SVIDs Bundles from a Workload API
+
+## SPIFFE Workload API ClientExample
+
+The `WorkloadAPIClient` provides the method `fetchX509SVIDs` that fetches SVIDs from the Workload API,
+using a BlockingStub and an very basic ExponentialBackoff Policy.
+
+The channel is configured based on the Address (tcp or unix socket) and the OS detected.
+
+## Use
+```
+WorkloadAPIClient workloadAPIClient = new WorkloadAPIClient("/tmp/agent.sock");
+List<X509SVID> svids = workloadAPIClient.fetchX509SVIDs();
+for (X509SVID svid : svids) {
+    System.out.println("Spiffe ID fetched " + svid.getSpiffeId());
+}
+```
```

Comment:
A real-world example would be a lot more useful here I think. How can I set a callback? How can I wire it up to my TLS listener? An example showing how to implement rotation is important IMO.